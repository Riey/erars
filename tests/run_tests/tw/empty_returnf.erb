@SYSTEM_TITLE
PRINTFORML INT_FUNC()=`{INT_FUNC()}` STR_FUNC()=`%STR_FUNC()%`

@INT_FUNC
#FUNCTION
RETURNF

@STR_FUNC
#FUNCTIONS
RETURNF
