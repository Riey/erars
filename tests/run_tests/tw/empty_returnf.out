INT_FUNC()=`0` STR_FUNC()=``
